publication-commons/src/main/java/no/unit/nva/publication/model/business/importcandidate/NotApplicable.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.publication.model.business.importcandidate;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.Username;
+import nva.commons.core.JacocoGenerated;
+
+import java.time.Instant;
+import java.util.Objects;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class NotApplicable implements ImportStatus {
+
+    private final static String SET_TO_NOT_APPLICABLE_BY = "setToNotApplicableBy";
+    private final static String COMMENT = "comment";
+    private final static String SET_TO_NOT_APPLICABLE_DATE = "setToNotApplicableDate";
+
+    @JsonProperty(SET_TO_NOT_APPLICABLE_BY)
+    private final Username setToNotApplicableBy;
+
+    @JsonProperty(COMMENT)
```

Comment:
Comment heter merknad i skissene. 

I følge Terje så kan vi gjøre det enkelt i første omgang med å ha det kun som en tekst streng. Så hvis importǿrene klager og ønsker seg mer funksjonalitet så lager vi det etterhvert. 

Merknadsfeltet eksisterte ikke i gammel versjon av sentralimport.

Reply:
Enig ja, vi gjør det så enkelt som mulig

Reply:
Since this is stored in a document base, you can call your fields whatever...
However, there is no point in using interfaces or inherency if field names are different for the same information.
You want it to be the same, so you don't have to write the same code over and over again, just with different names.